web/pdf_history.js

```diff
@@ -33,26 +35,13 @@ const UPDATE_VIEWAREA_TIMEOUT = 1000; // milliseconds
  * @property {EventBus} eventBus - The application event bus.
  */
 
-/**
```

Comment:
Please revert all changes in this file, since it really doesn't seem great having to look in a different file to find the docs.

Reply:
I was worried about the duplicate definitions diverging over time. Happy to revert though

Reply:
I suppose the types CI will catch incompatible definitions